crypto/errors.go

```diff
@@ -1,7 +1,15 @@
 package crypto
 
 import (
+	"errors"
 	"fmt"
 )
 
 var errInvalidSignatureReturned = fmt.Errorf("ed25519 library returned an invalid signature")
```

Comment:
If all the rest of the error are defined using `errors.New`, then this one should be defined using `errors.New` as well.